examples/cpp/01_add_vectors/addVectors.okl

```diff
@@ -2,7 +2,7 @@
                         const float *a,
                         const float *b,
                         float *ab) {
-  for (int i = 0; i < entries; ++i; @tile(16, @outer, @inner)) {
+  for (int i = 0; i < entries; ++i; @tile(4, @outer, @inner)) {
```

Comment:
Is there a reason we want to block by 4 instead of 16? This feels like a very tiny warp size for GPU modes

Reply:
This likely got changed during testing to ensure more than one work-group is launched since the total entries for the example is only 12. The tile size of 16 (or large) makes more sense, but then we should change the number of entries to 128 or larger also.